core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -78,4 +78,31 @@
 # once a day on the 1st and 15th of every month except December
 H H 1,15 1-11 *
 </pre>
+<p>
+  <b>Timezones specification</b>. <br/>
+  Starting from 1.615 Jenkins supports the specification 
+  of different timezones in this field. It allows to natively configure all
+  periodic operations.
+  
+  <ul>
+    <li>Timezone can be specified in the first line, which should contain the &quot;TZ=$(timezoneID)&quot; string</li>
```

Comment:
@daniel-beck I think that's covered by "This behaviour *can* be changed" above.

But the first line here could also be changed to "If a timezone is specified, it must be on the first line of this field, in the format..."